styles/cart.css

```diff
@@ -0,0 +1,64 @@
+table,
+th {
+  text-align: center;
+  border-bottom: 1px solid var(--gray-300);
+}
+th,
+tr,
+td {
+  padding: 0.5rem;
+}
+
+.cart-page {
+  padding: 2rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+.shop-link {
+  margin-top: 1rem;
+}
+
+.horizontal {
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+}
+.hori-stats {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  gap: 1rem;
+}
+.my-cart {
+  margin-top: 1.5rem;
+  display: flex;
+}
+
+.qty-input {
+  padding: 0.5rem;
+  width: 3.5rem;
+  border: 1px solid var(--gray-500);
+  outline: none;
```

Comment:
things like no outline on input is common for every input component so u make so u can add  make a common input class and add such styles there if possible ?

Reply:
yes it will be good, Thanks for that I'll look into it.